Eval.Core/Selection/Parent/TournamentParentSelectionOld.cs

```diff
@@ -18,7 +18,7 @@
 namespace Eval.Core.Selection.Parent
 {
     [Obsolete("Use new tournament selection instead")]
```

Comment:
Since we already have breaking changes, might as well remove obsolete code

Reply:
Agree